package.json

```diff
@@ -41,7 +41,9 @@
     "lint-ci": "concurrently \"eslint src/ --quiet\" \"stylelint '*/**/*.css' --quiet\" \"htmlhint\" \"prettier --c './src/public/**/*.html' --ignore-path './dist/**'\"",
     "version": "auto-changelog -p && git add CHANGELOG.md",
     "prepare": "husky install",
-    "pre-commit": "lint-staged"
+    "pre-commit": "lint-staged",
+    "storybook": "npm run --prefix frontend storybook",
+    "postinstall": "cd frontend && npm install"
```

Comment:
this postinstall script will probably conflict with the postinstall in `develop` haha, but boat reach bridge head automatically straight